api/utils_test.go

```
@@ -0,0 +1,47 @@
+
+	_, err := (StaticToken{}).Authorization(context.Background(), OperationName("test"))
+	if err == nil {
+		t.Fatal("Authorization() error = nil, want an error")
+	}
+}
+
+func TestStaticTokenAuthorizationProducesBearerHeader(t *testing.T) {
+	t.Parallel()
+
+	client := &Client{sec: StaticToken{Token: "Bearer token-value"}}
+	req, err := http.NewRequest(http.MethodGet, "https://example.test", nil)
```

golangci-lint (2.13.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)